liblangutil/SemVerHandler.cpp

```
@@ -166,6 +166,7 @@ SemVerMatchExpression SemVerMatchExpressionParser::parse()
 	catch (SemVerError const&)
 	{
 		reset();
+		return nullopt;
```

We could also rethrow the exception, instead of returning an option.

---

Fine with option